src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -500,11 +500,18 @@ function MoneyRequestView({
                                     getErrorForField('tag', {
                                         tagListIndex: index,
                                         tagListName: name,
+                                        tagListValue: TransactionUtils.getTagForDisplay(transaction, index),
+                                        policyHasDependentTags: PolicyUtils.hasDependentTags(policy, policyTagList),
                                     })
                                         ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR
                                         : undefined
                                 }
-                                errorText={getErrorForField('tag', {tagListIndex: index, tagListName: name})}
+                                errorText={getErrorForField('tag', {
+                                    tagListIndex: index,
+                                    tagListName: name,
+                                    tagListValue: TransactionUtils.getTagForDisplay(transaction, index),
```

Comment:
We have tagListIndex, and tagListName, why do we need tagListValue?

Reply:
@cead22 I need to check if the tag is already filled, if so I need to remove the `ALL_TAG_LEVELS_REQUIRED` from showing for this tag